docs2/site/docs/migrations/migration5.md

```diff
@@ -137,7 +137,7 @@ Custom attributes can also be added to perform the following behavior changes:
 - Override detected nullability or list nullability state
 - Override chosen underlying graph type
 
-#### Method support for `AutoRegisteringObjectGraphType<T>` instances
+#### Method and argument support for `AutoRegisteringObjectGraphType<T>` instances
```

Comment:
All this paragraph looks like a separate section of the main documentation.

Reply:
Yes, it is on my todo list to update the main documentation - see #2893 

We can delete it out of here if desired at that time.

Reply:
(I want to finish building some other relevant functionality first.)

Reply:
👍